storage/innobase/xtrabackup/test/t/bug1192834-purge.sh

```diff
@@ -0,0 +1,33 @@
+##########################################################################
+# Bug 1192834: Crash during apply with index compaction enabled          #
+##########################################################################
+
+. inc/common.sh
+
+skip_test "Enable when bug #1192834 is fixed in 2.4"
```

Comment:
If I get it right, this PR would fix the 1192834. Should we enable this test now?